public-ui/src/app/views/ehr-view/ehr-view.component.ts

```diff
@@ -61,6 +61,7 @@ export class EhrViewComponent implements OnInit, OnDestroy {
   showGenderGraph = false;
   showAgeGraph = true;
   showSourcesGraph = false;
+  showTopConcepts = {};
```

Comment:
I'm not sure about the UX of making this sticky per domain. Seems like a single boolean would make more sense.

Reply:
makes more sense. thanks.